packstream/packstream-specification-1.md

```diff
@@ -781,16 +781,21 @@ DateTime::Structure(
 )
 ```
 
-- The `seconds` are seconds since the adjusted [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)). This is not [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time).
 - The `tz_offset_seconds` specifies the offset in seconds from UTC.
+- The `seconds` are the above offset added to the seconds since the [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)).
```

Comment:
I don't understand this description either :/

How about something along the lines of

> - The `seconds` and `nanoseconds` are the time elapsed since `1970-01-01 00:00:00` in the same timezone.

Alternatively

> - `seconds` and `nanoseconds` are computed by replacing (*not converting*) the timezone information of a given DateTime with `UTC` and computing its difference from (i.e., time elapsed since) [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)) (`1970-01-01T00:00:00Z`).